plugins/utilities.json

```diff
@@ -3,6 +3,101 @@
   "description": "Utilities",
   "plugins_base_url": "https://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugins/utilities",
   "plugins": {
+    "sandbox": {
+      "description": "Spawn, control and bots, change music, add teams, spawn objects, change game values in real time, and much more!",
+      "external_url": "",
+      "authors": [
+        {
+          "name": "BrotherBoard",
+          "email": "thehero2012008@gmail.com",
+          "discord": "BrotherBoard"
+        }
+      ],
+      "versions": {
+        "1.2.0": {
+          "api_version": 8,
+          "commit_sha": "802fada",
+          "released_on": "18-05-2024",
+          "md5sum": "ce6e2acb7dca2d41df1d5b5dcb0064a1"
+        }
+      }
+    },
+    "updown": {
+      "description": "Adds UP and DOWN buttons in party window to recall messages in chat, just like linux terminal lol",
```

Comment:
i would still prefer to not have the last bit since description are supposed to convey information not emotions but i leave it upon you to decide

Reply:
so it's that serious